### PR TITLE
Use supports_vm_import? instead of validate_import_vm

### PR DESCRIPTION
**Pull request by smelamud**
Depends-on: https://github.com/ManageIQ/manageiq/pull/16526

### CODE REVIEW

**Comment by oourfali** [comment]
@miq-bot assign @borod108

File: app/models/manageiq/providers/redhat/infra_manager.rb [hunk 1]
```diff
@@ -22,6 +22,10 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
 
   supports :provisioning
   supports :refresh_new_target
+  supports :vm_import do
```
Comment (borod108) [inline app/models/manageiq/providers/redhat/infra_manager.rb]: As far  as  I know the proper way to use the :supports method is with unsupported_reason_add, please see an example here:
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb#L5

Comment (smelamud) [reply app/models/manageiq/providers/redhat/infra_manager.rb]: Done.

File: app/models/manageiq/providers/redhat/infra_manager.rb [hunk 2]
```diff
@@ -22,6 +22,10 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
...
+    # The version of the RHV needs to be at least 4.1.5 due to https://bugzilla.redhat.com/1477375
+    version_higher_than?('4.1.5')
```
Comment (masayag) [inline app/models/manageiq/providers/redhat/infra_manager.rb]: note that this method was recently renamed https://github.com/ManageIQ/manageiq-providers-ovirt/pull/143#pullrequestreview-79021137

Comment (smelamud) [reply app/models/manageiq/providers/redhat/infra_manager.rb]: Done.

File: app/models/manageiq/providers/redhat/infra_manager/vm_import.rb [hunk 3]
```diff
@@ -77,7 +72,7 @@ def submit_configure_imported_vm_networks(userid, vm_id)
   def check_import_supported!(source_provider)
     raise _('Cannot import archived VMs') if source_provider.nil?
 
-    raise _('Cannot import to a RHEV provider of version < 4.1.5') unless validate_import_vm
+    raise _('Cannot import to a RHEV provider of version < 4.1.5') unless supports_vm_import?
```
Comment (borod108) [inline app/models/manageiq/providers/redhat/infra_manager/vm_import.rb]: I believe this should be: unsupported_reason(:vm_import) unless supports_vm_import?

Comment (smelamud) [reply app/models/manageiq/providers/redhat/infra_manager/vm_import.rb]: Done.

File: app/models/manageiq/providers/redhat/infra_manager.rb [hunk 4]
```diff
@@ -22,6 +22,10 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
 
   supports :provisioning
   supports :refresh_new_target
+  supports :vm_import do
+    # The version of the RHV needs to be at least 4.1.5 due to https://bugzilla.redhat.com/1477375
+    unsupported_reason_add(:vm_import, _('RHV version 4.1.5+ required')) unless version_at_least?('4.1.5')
```
Comment (masayag) [inline app/models/manageiq/providers/redhat/infra_manager.rb]: @smelamud  would you like to maintain the previous message "Cannot import to a RHV provider of version < 4.1.5" ? so both action (vm import) and requirement (RHV > 4.1.5) are included ? 

(not insisting on it)

Comment (smelamud) [reply app/models/manageiq/providers/redhat/infra_manager.rb]: @masayag Yes, it is better message indeed. Done.

**Comment by smelamud** [comment]
@miq-bot add_labels gaprindashvili/yes

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit ebaffa010dcaa49a697ef3553015d2381801bc05
Author: Moti Asayag <masayag@redhat.com>
Date:   Mon Nov 27 16:51:59 2017 +0200

    Merge pull request #154 from smelamud/supports-vm-import
    
    Use supports_vm_import? instead of validate_import_vm
    (cherry picked from commit afa9354c961fc74cc3d891eb9a869082f322e22d)
```

**Pull request merged**